src/index.html

```diff
@@ -17,6 +17,81 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="container">
+        <a href="#"
+          class="header__img"
+        >
+          <img
```

Comment:
Remember about correct indentation between parent and child elements. The `img` tag should be indented two spaces more than the `a` tag.